src/components/Card/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./Card.module.css";
+const back =
+  "https://images.unsplash.com/photo-1550537687-c91072c4792d?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=774&q=80";
+
+function Card({
+  image,
+  index,
+  off,
+  flip,
+  clickHandler,
+}: {
+  image: string;
+  index: number;
+  off: boolean;
+  flip: boolean;
+  clickHandler: (index: number, url: string) => void;
+}) {
+  return (
+    <div
+      onClick={() => {
+        clickHandler(index, image);
```

Comment:
Looks like you don't need `index` in this file other than here. You can define handler in the parent component and only pass `clickHandler` prop to `Card`.

Usually, such props are named `onClick`